sos/plugins/block.py

```diff
@@ -32,6 +41,9 @@ def setup(self):
             "ls -lanR /sys/block"
         ])
 
+        if self.get_option("all"):
```

Comment:
I just thought it would be a good idea to let it be optional, so the user can choose whether he wants to get all the data in /sys/block/ or not. I mean, in case the machine has lots of paths it would make this task slow, right?